packages/web-components/@sfx/product/stories/index.ts

```diff
@@ -0,0 +1,103 @@
+import { storiesOf } from '@storybook/html';
+import { withKnobs, text } from '@storybook/addon-knobs';
+import '../src/index.ts';
+import Product, { ProductModel } from '../src/product';
+
+const product = <ProductModel> {
+  name: 'Cool Product',
+  price: 39.99,
+  label: 'New Product',
+  promo: '25% off',
+  imageSrc: 'https://spudsmart.com/wp-content/uploads/2017/10/NZ-Bag-of-potatoes-cropped.png',
+  imageAlt: 'Good bag of potatoes'
+};
+
+storiesOf('Components|Product', module)
+  .addDecorator(withKnobs)
+  .add('Default', () => `
+    <sfx-product 
+      product="${text('Product Info', JSON.stringify({
```

Comment:
Is it possible to use `.product` here to allow for passing a complex object directly (instead of using `JSON.stringify()`)?

Reply:
Not without lit-html working, which we've put aside for now.